src/ocean/meta/types/Qualifiers.d

```diff
@@ -0,0 +1,199 @@
+/*******************************************************************************
+
+    Templates to define types with modified qualifiers based on some input
+    types.
+
```

Comment:
Perhaps mention that this is D2 transition related? (Well, mostly... `Unqual` is needed in D2 as well.)